### ISSUE TITLE
PHP 8 error "Required parameter $closure follows optional parameter $date"

### ISSUE THREAD

**Issue by ssang** [whitecube/nova-page#76]
/src/Pages/Template.php:359

```php
public function setDateIf($moment, $date = null, Closure $closure)
{
    if (!($date instanceof Carbon)) {
        $date = new Carbon($date);
    }

    if ($closure($date, $this->getDate($moment))) {
        return $this->setDate($moment, $date);
    }
}
```

**Comment by amidesfahani** [comment]
error still exists! any updates?

**Comment by Nyratas** [comment]
Hi, sorry for the delay. I just merged @martijngastkemper's PR which should fix this issue. Released under `v0.2.6`.
